PacketDotNet/TcpPacket.cs

```diff
@@ -294,12 +300,36 @@ virtual public bool CWR
             set { setFlag(value, TcpFields.TCP_CWR_MASK); }
         }
 
+        virtual public bool NS
+        {
+            get { return (AllFlags & TcpFields.TCP_NS_MASK) != 0; }
+            set { setFlag(value, TcpFields.TCP_NS_MASK); }
+        }
+
+        virtual public bool FlagA

```

Comment:
Not to my knowledge, I looked on wikipedia and the RFCs, and they are just labeled as reserved. I labeled them in order from LSB: A, B, and C. I needed to distinguish these bits, but as far as I know, there is no convention so far.